scroll-animations-1/Overview.bs

```diff
@@ -344,8 +345,8 @@ enum ScrollTimelineAutoKeyword { "auto" };
 dictionary ScrollTimelineOptions {
   Element? source = null;
   ScrollDirection orientation = "block";
-  DOMString start = "auto";
-  DOMString end = "auto";
+  (DOMString or ElementBasedOffset) start = "auto";
+  (DOMString or ElementBasedOffset) end = "auto";
```

Comment:
@majido Should this patch have updated the types of the `start` and `end` members of the `ScrollTimeline` interface too?

Reply:
Indeed! And thanks for the patch to fix it.